bigquery_etl/format_sql/tokenizer.py

```diff
@@ -410,7 +410,7 @@ class StatementSeparator(Token):
 class Operator(Token):
     """Operator."""
 
-    pattern = re.compile(r"<<|>>|>=|<=|<>|!=|.")
+    pattern = re.compile(r"<<|>>|>=|<=|=>|<>|!=|.")
```

Comment:
bigquery supports named arguments via `=>` now, as seen in https://cloud.google.com/bigquery/docs/reference/standard-sql/interval_functions#make_interval

Reply:
That's a really strange choice of syntax, although named arguments are a nice overall add.